lambda_code/src/models/constants.py

```diff
@@ -48,10 +72,10 @@ def is_urn_resource(s):
 
     @staticmethod
     def if_vaccine_not_give(not_given_flag):
-        if not not_given_flag or not_given_flag == Constants.vaccination_given_flag:
```

Comment:
To remove check for "empty" constant, as discussed in dev stand up.

Reply:
Separate bug ticket raised